.github/workflows/build.yaml

```diff
@@ -0,0 +1,71 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - dev-v*
+      - release-v*
+
+jobs:
+  build:
+    name: Validate
+    runs-on: org-${{ github.repository_owner_id }}-amd64-k8s
+    container: registry.suse.com/bci/bci-base:latest
+    steps:
+      - name: install dependencies
+        run: zypper --non-interactive install docker jq git make go awk
+      - name: install gh
+        run: |
+          mkdir -p /tmp/gh
+          curl -fsL https://github.com/cli/cli/releases/download/v2.63.2/gh_2.63.2_linux_amd64.tar.gz | tar xvzf - --strip-components=1 -C /tmp/gh
+          mv /tmp/gh/bin/gh /usr/bin/gh
+          chmod +x /usr/bin/gh
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Git safe directory
+        run: git config --global --add safe.directory "$PWD"
+      - name: Set up PATH
+        run: echo $PATH >> $GITHUB_PATH
+
+      - name: Checkout base branch
+        uses: actions/checkout@v4
+      - name: Checkout PR
+        run: gh pr checkout ${{ github.event.pull_request.number }}
+        env:
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout into branch
+        run: git checkout -b staging-pr-workflow
+
+      - name: Pull scripts
+        run: make pull-scripts
+
+      - name: Check release.yaml
+        run: make check-release-yaml
+
+      - name: Release PR Chekpoints
+        run: |
+          if [[ "${{ github.ref }}" == refs/heads/release-v* ]]; then
+            make validate-release-charts BRANCH=${{ github.event.pull_request.base.ref }} GH_TOKEN=${{ secrets.GITHUB_TOKEN }} PR_NUMBER=${{ github.event.pull_request.number }}
+            echo "Validating remote release branch"
+          fi
+
+      - name: Validate
+        run: |
+          if [[ "${{ github.ref }}" == refs/heads/release-v* ]]; then
+            echo "Validating remote release branch"
+            make validate remote=true
+          else
+            echo "Validating local branch"
+            make validate
+          fi
+
+      - name: Check container images
+        run: make check-images
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Are these creds still available through secrets on this repo?

Reply:
Yes they are!

They should not be?

Reply:
Other repos are using the EIO vault